Update multiple cluster doc (#2551)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_multiple_OpenSearch_clusters.md

   - *PWD*: The password to connect to OpenSearch.

>    - *DB*: The prefix for the OpenSearch indexes. The specified value should match across all the OpenSearch configurations in the *DBConfiguration.xml*

Reviewer: DBConfiguration.xml is no longer used from DataMiner 10.4.0/10.4.2 onwards, so I'd like to change this as follows:

```suggestion
   - *DB*: The prefix for the OpenSearch indexes. The specified value must be the same for each OpenSearch cluster defined in *DB.xml*.
```

   - **Password**: The password that will be used to connect to OpenSearch. This is the equivalent of the *PWD* tag in the *DB.xml* file (see [Specifying custom credentials for OpenSearch or Elasticsearch](xref:DB_xml#specifying-custom-credentials-for-opensearch-or-elasticsearch)).

>    - **Prefix**: The prefix for the OpenSearch indexes. This is the equivalent of the *DB* tag in the *DB.xml* file. The specified value should match across all the OpenSearch configurations in the *DBConfiguration.xml*. (see [Specifying a custom prefix for the indexes](xref:DB_xml#specifying-a-custom-prefix-for-the-indexes)).

Reviewer: ```suggestion
   - **Prefix**: The prefix for the OpenSearch indexes. This is the equivalent of the *DB* tag in the *DB.xml* file. The specified value must be the same for each OpenSearch cluster defined in *DBConfiguration.xml*. (see [Specifying a custom prefix for the indexes](xref:DB_xml#specifying-a-custom-prefix-for-the-indexes)).
```
